ChangeLog.d/add-aes-128bit-only.txt

```diff
@@ -0,0 +1,5 @@
+Features
+   * Add support for AES to only enable 128-bit key in order to save code size.
```

Comment:
Suggested rewording of the first sentence so it flows better: something like "Add support for enabling only AES-128 in order to reduce code size."

Reply:
I have combined both suggestions to rewrite changelog. Let me know if there are more questions.